packages/yourloops/pages/patient-view/target-and-alerts/target-and-alerts-view.tsx

```diff
@@ -0,0 +1,110 @@
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { FC, useEffect, useState } from 'react'
+import Container from '@mui/material/Container'
+import MonitoringAlertsContentConfiguration
+  from '../../../components/monitoring-alert/monitoring-alerts-content-configuration'
+import { Patient } from '../../../lib/patient/models/patient.model'
+import { MonitoringAlertsParameters } from '../../../lib/team/models/monitoring-alerts-parameters.model'
+import { usePatientsContext } from '../../../lib/patient/patients.provider'
+import { useAlert } from '../../../components/utils/snackbar'
+import { useTranslation } from 'react-i18next'
+import { useSelectedTeamContext } from '../../../lib/selected-team/selected-team.provider'
+import Card from '@mui/material/Card'
+import { useTheme } from '@mui/material/styles'
+import CardHeader from '@mui/material/CardHeader'
+import CardContent from '@mui/material/CardContent'
+import Typography from '@mui/material/Typography'
+import { useLocation } from 'react-router-dom'
+
+interface TargetAndAlertsViewProps {
+  patient: Patient
+}
+
+export const MONITORING_ALERTS_SECTION_ID = 'monitoring-alerts'
+
+export const TargetAndAlertsView: FC<TargetAndAlertsViewProps> = (props) => {
+  const { patient } = props
+  const theme = useTheme()
+  const { t } = useTranslation('yourloops')
+  const { updatePatientMonitoringAlertsParameters } = usePatientsContext()
+  const { selectedTeam } = useSelectedTeamContext()
+  const alert = useAlert()
+  const { pathname, hash, key } = useLocation()
+  const monitoringAlertsParameters = patient.monitoringAlertsParameters ?? selectedTeam.monitoringAlertsParameters
+
+  const [saveInProgress, setSaveInProgress] = useState<boolean>(false)
+
+  useEffect(() => {
+    if (hash === '') {
+      return
+    }
+
+    const sectionId = hash.replace('#', '')
+    const section = document.getElementById(sectionId)
+    if (section) {
+      section.scrollIntoView()
+    }
+  }, [pathname, hash, key])
+
+  const save = async (monitoringAlertsParameters: MonitoringAlertsParameters): Promise<void> => {
+    patient.monitoringAlertsParameters = monitoringAlertsParameters
+    setSaveInProgress(true)
+    try {
+      await updatePatientMonitoringAlertsParameters(patient)
+      alert.success(t('patient-update-success'))
+      setSaveInProgress(false)
+    } catch (error) {
+      console.error(error)
+      alert.error(t('patient-update-error'))
+      setSaveInProgress(false)
+    }
+  }
+
+  return (
+    <Container data-testid="target-and-alerts-container">
+      <Card variant="outlined" sx={{ padding: theme.spacing(2) }}>
+        <CardHeader title={t('target-and-alerts')} />
```

Comment:
I felt that this tab is feeling weird/unfinished/drafty and one thing that could help is to have the divider as in the design. Did you forget it?
![Screenshot from 2023-10-09 11-22-23](https://github.com/mdblp/blip/assets/6829894/26c3c79d-d1e9-498d-906a-842ed64028a4)

Reply:
I did 😬 Thanks for the sharp eyes